requirements/test.txt

```diff
@@ -1,5 +1,6 @@
 coverage
 codecov>=2.1
+nbval
```

Comment:
This is never run?

Reply:
That's for pytest's `--nbval-lax` option here:
https://github.com/PyTorchLightning/lightning-flash/blob/2317782c5ef485523b386487ca1f55ef90b19804/.azure-pipelines/gpu-tests.yml#L61

Reply:
That's for pytest's `--nbval-lax` option here:
https://github.com/PyTorchLightning/lightning-flash/blob/2317782c5ef485523b386487ca1f55ef90b19804/.azure-pipelines/gpu-tests.yml#L61